models/card.js

```diff
@@ -1,22 +1,27 @@
 module.exports = function(sequelize, DataTypes) {
-  var Card = sequelize.define("Card", {
```

Comment:
This line must be preserved according to Eli, as var card must be capitalized to var Card